docs/user-docs/latest/settings/managing-global-settings.md

```diff
@@ -10,10 +10,37 @@ Single Sign-On (SSO) can be made available on a Strapi application to allow admi
 
 To configure the SSO feature settings:
 
-1. Go to the *Global settings > Single Sign-On* area of the settings interface.
+1. Go to the *Global settings > Single Sign-On* sub-section of the settings interface.
 2. Define your chosen new settings:
 
 | Setting name      | Instructions                                                                                                                                                                                                                                                        |
 | ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Auto-registration | Click on **ON** to allow the automatic creation of a new Strapi administrator when an SSO login does not match an existing Strapi administrator account. If this setting is set on **OFF**, new Strapi administrators accounts must be created manually beforehand. |
 | Default role      | Choose among the drop-down list the role to attribute by default to auto-registrated Strapi administrators through SSO login.                                                                                                                                       |
+
+## Configuring Internationalization locales
+
+The [Internationalization plugin](/user-docs/latest/plugins/strapi-plugins.md#internationalization-plugin) allows to manage content in different languages, called "locales". Once the Internationalization plugin is installed in a Strapi application (see [Installing plugins via the Marketplace](../plugins/installing-plugins-via-marketplace.md)), administrators can manage locales from the *Global settings > Internationalization* sub-section of the settings interface.
+
+The *Internationalization* settings sub-section displays a table listing all locales available for the Strapi application. By default, only the English locale is configured and set as the default locale. 
+
+For each locale, the table displays the default ISO code of the locale, its optional display name and indicates if the locale is set as the default one. From the table, administrators can also:
+
+- Click on the edit button <Fa-PencilAlt /> to edit a locale
+- Click on the trash button <Fa-TrashAlt /> to delete a locale
+
+### Adding a new locale
+
+Administrators can add and manage as many locales as they want. There can however only be one locale set as the default one for the whole Strapi application.
+
+::: tip NOTE
+It is not possible to create custom locales. Locales can only be created based on the 500+ pre-created list of locales set by Strapi.
```

Comment:
[In the dev docs](https://github.com/strapi/documentation/pull/224/files#diff-0db7342d805c5a201d351b41ce44251488b1e91e61c43c31a90aebad267c013eR53), I plan to link to the [full list of locales](https://github.com/strapi/strapi/blob/features/i18n/packages/strapi-plugin-i18n/constants/iso-locales.json). Do you think it would be a good idea to add this link here as well?

Reply:
Yes it could definitely be helpful! I'm adding it.